### DISCUSSION TITLE
Positional encodings for molecular application

### DISCUSSION THREAD

**Discussion by pgniewko** [pyg-team/pytorch_geometric#6559] [Q&A] [answered]
Is there a community code on how to use positional encodings (RWPE or LEPE) on a molecular properties prediction set, like ZINC.

Also, is there any resource where I can learn how to implement a custom PE from scratch?

Thanks a lot!

**Comment by rusty1s** [comment, marked as answer]
The only example that makes use of positional encodings can be found in https://github.com/pyg-team/pytorch_geometric/blob/master/examples/graph_gps.py, but it is not yet performing that well :(
